byway/src/main/webapp/base.js

```diff
@@ -15,6 +16,23 @@ function setProgressBar(pageNumber){
   document.getElementById('view-route-progress-bar').href = configureTripKeyForPath(tripKey, '/routepage.html');
 }
 
+function setLogoutLink(){
```

Comment:
Since this also redirect when the user is *not* logged in, you may want to choose a different name. Keep in mind that `set` usually means 'assign to a field', where this is more like 'setup logout link'; set = simple assignment, setup = may do some other work. Also, since this does a redirect, it should have a doc comment that mentions that.

Reply:
fixed